functorch/_src/eager_transforms.py

```diff
@@ -754,7 +771,13 @@ def jvp(func, primals, tangents, *, strict=False):
                 partial(_undo_create_differentiable, level=level), primals_out)
             tangents_out = tree_map(
                 partial(_undo_create_differentiable, level=level), tangents_out)
-            return tree_unflatten(primals_out, spec), tree_unflatten(tangents_out, spec)
+
+            primals_out_unf = tree_unflatten(primals_out, spec)
+            tangents_out_unf = tree_unflatten(tangents_out, spec)
```

Comment:
nit: primals_out_unflatten is more verbose but it's easier to understand than `unf`